phplint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#Author Addshore
+
+find ./ -type f -name '*.php' -exec php -l {} \; | grep "Errors parsing ";
```

Comment:
consider using standard mediawiki tools (codesniffer, phan) instead

Reply:
This would likely be done as an entirely different PR, as it should be done for all our extensions as well probably.